src/prefect/server/api/root.py

```diff
@@ -11,3 +13,18 @@
 async def hello():
     """Say hello!"""
     return "👋"
+
+
+@router.get("/ready")
+async def peform_readiness_check(
+    db: PrefectDBInterface = Depends(provide_database_interface),
+    response: Response = None,
+):
+    is_db_connectable = await db.is_db_connectable()
+
+    if is_db_connectable:
+        response.status_code = status.HTTP_200_OK
+        return is_db_connectable
+
+    response.status_code = status.HTTP_500_INTERNAL_SERVER_ERROR
+    return "Database is not available"
```

Comment:
opted not to return a JSONResponse instance, since this is just information for an operator to know why a readiness check might have failed

Reply:
IMO, JSON responses are preferable because we can extend them without breaking API compatibility. Sadly we don't do this consistently.

Reply:
@parkedwards Can you return a 503 with this and a JSON response like `{"reason": "Database is not available"}`?